akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/FlightRecording.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+import java.nio.file.Path
+
+import akka.actor.{ ActorSystem, ExtendedActorSystem }
+
+/**
+ * This will work on JDK11 and JDK8 built with the enable-jfr flag (8u262+).
+ *
+ * For Akka JRF recordings you may need to run a publish for multi jvm tests
+ * to get the ComileJDK9 things compiled.
+ */
+class FlightRecording(system: ActorSystem) {
```

Comment:
Belongs in the testkit perhaps (possibly InternalApi though)?

Reply:
Perhaps we wait until we use it in multiple places?